src/applications/vaos/components/calendar/CalendarCell.jsx

```diff
@@ -11,7 +11,10 @@ const CalendarCell = ({
 }) => {
   if (date === null) {
     return (
-      <button className="vaos-calendar__calendar-day vads-u-visibility--hidden" />
+      <div
```

Comment:
Early testing with VoiceOver is still missing this cell. Can we refactor it to hide an empty button inside the cell? Seems overly prescriptive, but it does announce properly, and should follow suit in the other screen readers.

```html
<div role="cell" class="vaos-calendar__calendar-day">
  <button class=" vads-u-visibility--hidden"></button>
</div>
```